src/common_utils/database_preparing/csv_to_database_filler.py

```diff
@@ -0,0 +1,74 @@
+from src.common_utils.database_service import DatabaseProxy
+import src.common_utils.custom_exceptions as exceptions
+import csv
+from src.common_utils.language_utils.sentence_filter_utils import SentenceFilter
+from csvWriter import CsvWriter
+import os.path
+
+def initialize_main_collection_from_scrapper(db):
+    # to prepare necessary files from one file: DB_FINAL_<number>.csv
```

Comment:
if you mark method description with triple quotes IDE recognizes it and shows during usage :)